### DISCUSSION TITLE
3.X版本不再支持快应用了吗？

### DISCUSSION THREAD

**Discussion by purvisdevgit** [NervJS/taro#13314] [Feature Request]
Taro3.X版本不再支持快应用了吗？以后还有没有可能加上？

**Comment by Chen-jj** [comment]
快应用使用者目前看太少了，实现的 ROI 比较低，官方应该不会跟进了，如果社区有意愿实现的话可以提供帮助

**Comment by purvisdevgit** [comment]
好的谢谢

发自我的小米
在 chenjiajian ***@***.***>，2023年2月16日 下午5:44写道：

快应用使用者目前看太少了，实现的 ROI 比较低，官方应该不会跟进了，如果社区有意愿实现的话可以提供帮助

**Reply by esymeptoo** [reply]
不支持了麻烦官网的文档和模板生成的scripts启动命令都删除一下关于quickapp的内容，容易产生误导
